tests/testthat/test-vctr.R

```diff
@@ -92,6 +92,19 @@ test_that("equality functions remapped", {
   expect_true(anyDuplicated(x))
 })
 
+test_that("is.na<-() and new_vctr() commute", {
+  x <- new_vctr(1:4)
+  y <- 1:4
+
+  expect_error(is.na(x) <- c(FALSE, FALSE, TRUE, NA), NA)
```

Comment:
Why is this an error?

Reply:
It's not. Fixed here, added an issue to handle other instances of this anti-pattern.